client/components/jetpack/connection-health/index.tsx

```diff
@@ -98,8 +98,10 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		return (
 			<ErrorNotice
 				errorType={ errorType }
-				errorText={ translate( 'Jetpack encounters REST API connection issues.' ) }
-				noticeActionHref={ localizeUrl( 'https://wordpress.com/support/resolve-jetpack-errors/' ) }
+				errorText={ translate( 'REST API is disabled.' ) }
```

Comment:
```suggestion
				errorText={ translate( 'Jetpack can’t communicate with your site because the REST API is not responding correctly.' ) }
```

Reply:
@danielbachhuber the docs say the message I added. Is this okay to have a different message? Because as I read them, I except to see the same message, which is not the case.

Reply:
@kozer Which message do you think is more clear? It's easy enough to update the docs.

Reply:
Yours! What's the process of updating the docs? I'll take a look in the field guide